zebra-chain/src/transaction/tests/vectors.rs

```diff
@@ -105,6 +106,7 @@ fn empty_v5_round_trip() {
     zebra_test::init();
 
     let tx = Transaction::V5 {
+        consensus_branch_id: Nu5.branch_id().expect("nu5 has a branch id"),
```

Comment:
Let's turn this into a constant, rather than repeating it 5 times.

Remember, there will be other network upgrades after NU5, and we'll want to update the tests for them.